wear/src/main/java/org/gdg/frisbee/android/ColorConfigActivity.java

```diff
@@ -61,16 +61,9 @@ public void onCentralPositionChanged(int i) {
     private void displayColorSelections() {
         List<String> colorList = new ArrayList<>();
 
-        if (mHeader.contains("Background")) {
-            String[] backgrounds = getResources().getStringArray(R.array.background_selection);
```

Comment:
I re-enabled background configuration. Changed background selection from dark/white to the same color selection for the others.